integrationTests/singleShard/transaction/interceptedBulkTx/interceptedBulkTx_test.go

```diff
@@ -17,6 +17,7 @@ import (
 )
 
 var stepDelay = time.Second
+var p2PBootstrapDelay = time.Second * 5
```

Comment:
integrationTests.P2pBootstrapDelay instead 5 * time.Second or delete this var and use directly integrationTests.P2pBootstrapDelay everywere in this file?

Reply:
done